From 602d90e7b04d39576d458094caf031b9638d4c9c Mon Sep 17 00:00:00 2001
From: Awais Belal <awais_belal@mentor.com>
Date: Tue, 29 Mar 2016 15:10:02 +0500
Subject: [PATCH] allow building glx backend without x11 enabled

The X11 requirements are dealt through the base package
so we allow this package to build glx without x11 enabled.

Signed-off-by: Awais Belal <awais_belal@mentor.com>
---
 configure.ac | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/configure.ac b/configure.ac
index 8dde9ca..65aad67 100644
--- a/configure.ac
+++ b/configure.ac
@@ -230,7 +230,7 @@ AM_CONDITIONAL(USE_X11, test "$USE_X11" = "yes")
 
 # Check for GLX
 USE_GLX="no"
-if test "$USE_X11:$enable_glx" = "yes:yes"; then
+if test "$enable_glx" = "yes"; then
     PKG_CHECK_MODULES([GLX], [gl x11], [USE_GLX="yes"], [:])
     saved_CPPFLAGS="$CPPFLAGS"
     saved_LIBS="$LIBS"
-- 
1.9.1

